Ethereal Artefacts/Ethereal Artefacts/Views/Components/SearchField.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  Search.swift
+//  Ethereal Artefacts
+//
+//  Created by Albert Khurshudyan on 8.05.23.
+//
+
+import SwiftUI
+
+struct SearchField: View {
```

Comment:
There is a Search Bar that is build in in SwiftUI you can have a look ".searchable"